spark/homescreens.json

```diff
@@ -0,0 +1,45 @@
+{
+  "homescreens": [
+    [
+      ["apps", "communications", "dialer"],
+      ["apps", "sms"],
+      ["apps", "email"],
+      ["apps", "search"],
+      ["apps", "gallery"],
+      ["apps", "camera"],
+      ["apps", "marketplace.firefox.com"],
+      ["apps", "clock"],
+      ["apps", "settings"]
+    ], [
+      ["outoftree_apps", "directory"],
+      ["outoftree_apps", "studio"],
+      ["outoftree_apps", "bugillalite"],
```

Comment:
the preload tool creates the folder based on the manifest name (which currently has a typo), I think the behaviour of the preload tool should be changed but for now this is correct.